compiler/src/dmd/initsem.d

```diff
@@ -1017,6 +1024,12 @@ Initializer inferType(Initializer init, Scope* sc)
         return new ErrorInitializer();
     }
 
+    Initializer visitDefault(DefaultInitializer i)
+    {
+        error(i.loc, "cannot infer type from default initializer");
```

Comment:
```suggestion
        error(i.loc, "could not infer type from default initializer");
```
Or even better "A type could not be inferred from"

Reply:
I don't see an improvement?